### PR TITLE
update module dependencies to latest version

### PR DESCRIPTION
**Pull request by humblec**
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -5,23 +5,23 @@ go 1.18
...
+	github.com/prometheus/client_golang v1.13.1
+	github.com/prometheus/client_model v0.3.0
+	github.com/prometheus/common v0.37.0
+	github.com/spf13/cobra v1.6.1
+	google.golang.org/grpc v1.50.1
+	k8s.io/api v0.25.3
+	k8s.io/apimachinery v0.25.3
+	k8s.io/client-go v1.5.2
+	k8s.io/component-base v0.25.3
+	k8s.io/component-helpers v0.25.3
```
Comment (xing-yang) [inline go.mod:24]: This one is missed:
```
k8s.io/kubernetes v1.25.2
```

Comment (humblec) [reply go.mod:24]: updated that too @xing-yang

File: go.mod [hunk 2]
```diff
@@ -5,25 +5,25 @@ go 1.18
...
-	k8s.io/component-base v0.25.2
-	k8s.io/component-helpers v0.25.2
+	github.com/prometheus/client_golang v1.13.1
+	github.com/prometheus/client_model v0.3.0
+	github.com/prometheus/common v0.37.0
+	github.com/spf13/cobra v1.6.1
+	google.golang.org/grpc v1.50.1
+	k8s.io/api v0.25.3
+	k8s.io/apimachinery v0.25.3
+	k8s.io/client-go v1.5.2
```
Comment (xing-yang) [inline go.mod:22]: @humblec Can you fix this?  This is an old version.

Comment (xing-yang) [reply go.mod:22]: I submitted a PR: https://github.com/kubernetes-csi/external-snapshotter/pull/785

**Pull request merged**